lib/index.js

```diff
@@ -224,10 +226,86 @@ function getPolyfillString(options) {
 	;
 }
 
+function getDetectString(options) {
```

Comment:
Isn't this kind of just reimplementing `getPolyfillString`?  Could the existing `getPolyfillString` be abstracted into something like `buildOutputString` and then be called in two different modes?  You could also avoid adding the `co` dependency`.  I'm not against using third party code but if we already have a way of doing something in one part of the code it seems like a good idea to be consistent with that.